users/thystips/thystips.h

```diff
@@ -0,0 +1,126 @@
+
+/* Copyright 2021 Jonavin Eng @Jonavin
+ * Copyright 2022 Antoine Thys <contact@thystips.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// DEFINE MACROS
+#define ARRAYSIZE(arr) sizeof(arr) / sizeof(arr[0])
```

Comment:
Why this over the [built in](https://github.com/qmk/qmk_firmware/blob/master/quantum/util.h#L47) macro?